pkg/tests/cross-tests/rapid_test.go

```diff
@@ -62,6 +62,45 @@ func TestDiffConvergence(outerT *testing.T) {
 	})
 }
 
+func TestCreateInputsConvergence(outerT *testing.T) {
+	_, ok := os.LookupEnv("PULUMI_EXPERIMENTAL")
+	if !ok {
+		outerT.Skip("TODO - we do not currently pass all cases; using this as an exploration tool")
+	}
+	outerT.Parallel()
+
+	log.SetOutput(io.Discard)
+	tvg := &tvGen{
```

Comment:
Can we please name this more descriptively? Unless there's a convention I'm not aware of, it would be very helpful in understanding this code when reading.